ntbs-service/Models/SocialContextVenue.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using ExpressiveAnnotations.Attributes;
+using ntbs_service.Models.Enums;
+using ntbs_service.Models.Interfaces;
+using ntbs_service.Models.Validations;
+
+namespace ntbs_service.Models
+{
+    public class SocialContextVenue : ModelBase, IHasPostcode
+    {
+        public int SocialContextVenueId { get; set; }
+
+        public int NotificationId { get; set; }
+        // public virtual Notification Notification { get; set; }

```

Comment:
will remove this and add the same comment as for ManualTestResult about not wanting to validate the whole model!